internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -247,10 +238,7 @@ func convertMountPoints(input map[string]manifest.Volume) ([]*template.MountPoin
 	}
 	var output []*template.MountPoint
 	for name, volume := range input {
-		mp, err := convertMountPoint(aws.String(name), volume.ContainerPath, volume.ReadOnly)
-		if err != nil {
-			return nil, err
-		}
+		mp := convertMountPoint(aws.String(name), volume.ContainerPath, volume.ReadOnly)
```

Comment:
nit: this is a preference, but I really like the [inline variable refactor](https://refactoring.com/catalog/inlineVariable.html) to reduce the size of the fn body
```suggestion
		output = append(output,  convertMountPoint(aws.String(name), volume.ContainerPath, volume.ReadOnly))
```